### DISCUSSION TITLE
How to move Laragon project to another computer?

### DISCUSSION THREAD

**Discussion by rahmaazs** [leokhoa/laragon#445] [Q&A]
I already have my laragon projects (wordpress) in my laptop. Can i move all these my laragon projects to my another laptop? If yes, how do i do it? I really have no clue about it. Please help me. Thank you!

**Comment by leokhoa** [comment]
@rahmaazs : Yes, you can as Laragon is very portable:
Ref: https://laragon.org/docs/portable.html

**Reply by rahmaazs** [reply]
do you have a tutorial video or an article that explains how to move laragon projects to another computer?
thank you!

**Reply by promatik** [reply]
Hi @rahmaazs, depending on your OS, all Laragon is placed in a single folder, in my case, I'm in windows, so it's in "C:\laragon".

You can move the entire folder, or if you want to move just the project, it's inside "C:\laragon\www".

**Reply by rahmaazs** [reply]
My OS is also Windows. 
So, if i want to move my projects to another device, i need to copy the entire laragon folder in "C:\laragon" and paste it into "C:\laragon" folder in my other device. Is that right?

**Comment by arun07as** [comment]
To move your WordPress project, you will need to do two things,

1. Copy your project ( if you use the default settings, it will be in `C:\laragon\www\your-project-name` ) or make an archive ( zip/rar ) and paste it on the other laptop
2. Backup your database and restore the backup on the other laptop

For most sites, these two steps should be enough. If your website requires other software to be installed on the system, make sure you install those on the other laptop as well

**Reply by zkmark** [reply]
> To move your WordPress project, you will need to do two things,
> 
> 1. Copy your project ( if you use the default settings, it will be in `C:\laragon\www\your-project-name` ) or make an archive ( zip/rar ) and paste it on the other laptop
> 2. Backup your database and restore the backup on the other laptop
> 
> For most sites, these two steps should be enough. If your website requires other software to be installed on the system, make sure you install those on the other laptop as well

Is there any way to migrate the paths of applications like node/git and others?
So that those applications are recognized by windows

**Comment by izmeez** [comment]
Is it necessary to edit the laragon.ini file, the vhosts and the my.ini files to remove the drive identifier such as C: ?

**Comment by leokhoa** [comment]
@izmeez : Normally, Laragon will automatically adapts those for you so you don't need to do anything.

**Comment by izmeez** [comment]
@leokhoa Thank you for the prompt answer. Also, thank you for creating the Laragon project. I am new to using it and so far I am very impressed with it.

**Comment by leokhoa** [comment]
@izmeez: You are welcome :)
